packages/aws-cdk/lib/init-templates/app/csharp/src/HelloCdk/HelloStack.cs

```diff
@@ -20,7 +21,7 @@ public HelloStack(Construct parent, string id, IStackProps props) : base(parent,
                 DisplayName = "My First Topic Yeah"
             });
 
-            topic.SubscribeQueue(queue, null);
+            topic.AddSubscription(new SqsSubscription(queue, null));
```

Comment:
what's this `null`?

Reply:
I believe .NET does not support optional arguments